doc/newsfragments/2412_new.customizable_sigint_timeout_for_drivers.rst

```diff
@@ -0,0 +1 @@
+Added sigint_timeout to drivers that call kill_process
```

Comment:
Some reformatting is due, please enclose sigint_timeout and kill process in ````

Reply:
Okay sure, well noted